release-notes/opensearch-observability.release-notes-1.3.0.0.md

```diff
@@ -0,0 +1,95 @@
+* Revert query pre-processing for parse command ([#497](https://github.com/opensearch-project/observability/pull/497))
+* Fix create/edit page bug ([#475](https://github.com/opensearch-project/observability/pull/475))
+* Fix queries being filtered out ([#472](https://github.com/opensearch-project/observability/pull/472))
+
+### Documentation
+* Sync PPL commands doc with main repo ([#549](https://github.com/opensearch-project/observability/pull/549))
+* Fixed documentation links ([#534](https://github.com/opensearch-project/observability/pull/534))
+* Add parse command docs ([#535](https://github.com/opensearch-project/observability/pull/535))
+* Update Release Notes 1.2.4 ([#370](https://github.com/opensearch-project/observability/pull/370))
+* updating readme and badges ([#352](https://github.com/opensearch-project/observability/pull/352))
+
+### Maintenance
+* Change Default CI java version to 11 ([#504](https://github.com/opensearch-project/observability/pull/504))
+* Update backport and add auto-delete workflows ([#496](https://github.com/opensearch-project/observability/pull/496))
+* Add auto backporting functionality ([#491](https://github.com/opensearch-project/observability/pull/491))
+* bumping version to 1.2.2 ([#346](https://github.com/opensearch-project/observability/pull/346))
+* Update Workflow ([#360](https://github.com/opensearch-project/observability/pull/360))
+* bump main to 1.3 ([#361](https://github.com/opensearch-project/observability/pull/361))
+
+### Infrastructure
+* change to support java 8 in compile and runtime ([#575](https://github.com/opensearch-project/observability/pull/575))
+* Bump prismjs from 1.25.0 to 1.27.0 in /dashboards-observability ([#508](https://github.com/opensearch-project/observability/pull/508))
+* Update cypress test ([#564](https://github.com/opensearch-project/observability/pull/564))
+* fixed flaky panel test ([#565](https://github.com/opensearch-project/observability/pull/565))
+* Feature flyout tests ([#553](https://github.com/opensearch-project/observability/pull/553))
+* Add cypress tests for application analytics ([#544](https://github.com/opensearch-project/observability/pull/544))
+* Update panels cypress ([#545](https://github.com/opensearch-project/observability/pull/545))
+* Update cypress for trace analytics traces view ([#536](https://github.com/opensearch-project/observability/pull/536))
+* Cypress fix for panels and events ([#531](https://github.com/opensearch-project/observability/pull/531))
+* updated panels flaky jest tests ([#505](https://github.com/opensearch-project/observability/pull/505))
+* Guava package update ([#404](https://github.com/opensearch-project/observability/pull/404))
+* [main] jcenter removed from gradle.build ([#374](https://github.com/opensearch-project/observability/pull/374))
+* CVE fix:json-schema, gson & glob-parent ([#368](https://github.com/opensearch-project/observability/pull/368))
```

Comment:
cve and library bumps can be bug fixes. infrastructure is related to github workflows and any kind of tests

Reply:
i moved the ones that I though were library bumps but can you double check for me?

Reply:
@eugenesk24 sorry forgot to mention 575 adds older jdk to workflow for compatibility testing, you can leave it in infrastructure. And everything in maintenance except 361 can also be in infrastructure, i think maintenance is only for plugin version bump which maintains compatibility with core

Reply:
Ah got it, thank you so much. There should be some more documentation on what goes in what category